awkward-cpp/CMakeLists.txt

```diff
@@ -111,31 +111,21 @@ if(BUILD_TESTING)
 
 endif()
 
-option(PYBUILD "Build Python modules")
-cmake_dependent_option(AWKWARD_EXTERNAL_PYBIND11 "Build against an external pybind11" OFF
-                       "PYBUILD" OFF)
-
 # Third tier: Python modules.
-if(PYBUILD)
-  if(AWKWARD_EXTERNAL_PYBIND11)
-    find_package(pybind11 CONFIG REQUIRED)
-  else()
-    add_subdirectory(pybind11)
-  endif()
-
-  file(GLOB LAYOUT_SOURCES "src/python/*.cpp")
-  pybind11_add_module(_ext ${LAYOUT_SOURCES})
-  target_link_libraries(_ext PRIVATE awkward-static)
-
-  install(
-    TARGETS awkward awkward-parent awkward-cpu-kernels _ext
-    LIBRARY DESTINATION awkward
-    ARCHIVE DESTINATION awkward)
-
-  # Third tier: install without Python modules.
-else()
-  install(
-    TARGETS awkward-static awkward awkward-parent awkward-cpu-kernels awkward-cpu-kernels-static
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib)
-endif()
+find_package(pybind11 CONFIG REQUIRED)
+
+# Install python bindings
+file(GLOB LAYOUT_SOURCES "src/python/*.cpp")
+pybind11_add_module(_ext ${LAYOUT_SOURCES})
+target_link_libraries(_ext PRIVATE awkward-static)
+
+# Install pure-python files
+file(GLOB_RECURSE PYTHON_SOURCES "src/${SKBUILD_PROJECT_NAME}/*.py")
```

Comment:
If this is named correctly (which it looks like it is), the next version of scikit-build-core (feel free to bump the sha, already in master for matching named packages) will copy the files over for you. So `src/<package>` or `<package>` gets copied to `<package>`, similar to most other tools.